### ISSUE TITLE
Settings UI Is Not Visible

### ISSUE THREAD

**Issue by opticyclic** [microsoft/terminal#9134]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.19042.746]
Windows Terminal version: 1.6.10412.0
```

# Steps to reproduce

Start Windows Terminal Preview
Press `Ctrl-,`

# Expected behavior

The new Settings UI appears.

# Actual behavior

The settings json file appears

**Comment by WSLUser** [comment]
You need to add a keybinding in settings.json in order to open the Settings UI. The release notes (and docs) have the line entry you need to bind it. Because it's in Preview still, you can't use it by default.

**Comment by DHowett** [comment]
Yup!

**Comment by opticyclic** [comment]
FYI: It would have been nicer if you had copied the relevant part of the answer here (a la stackoverflow) or linked to the[ "release notes (and docs)"](https://github.com/microsoft/terminal/releases/tag/v1.6.10272.0).

TLDR:

> Until it's ready for prime time, it's hidden behind a NON-DEFAULT key binding. (#8048)
>To bind the settings UI to Ctrl+Shift+,, add the following to your keybindings/actions.

    { "command": { "action": "openSettings", "target": "settingsUI" }, "keys": "ctrl+shift+," },
